src/Faker/Provider/da_DK/Payment.php

```diff
@@ -16,4 +16,83 @@ public static function bankAccountNumber($prefix = '', $countryCode = 'DK', $len
     {
         return static::iban($countryCode, $prefix, $length);
     }
+
+    /**
+     * Returns an account number with between 7 and 10 digits
+     *
+     * @link https://amcbanking.com/kb/10540/
+     * @return string
+     */
+    public static function localBankAccountNumber()
```

Comment:
Please document all the new formatters to the locale-specific section of the documentation